tests/unit/call.ts

```diff
@@ -1034,7 +1034,7 @@ describe('Call', function() {
     beforeEach(() => {
       message = {
         content: { foo: 'foo' },
-        messageType: Call.MessageType.UserDefinedMessage,
+        messageType: 'UserDefinedMessage',
```

Comment:
Let's keep the real value 'user-defined-message' in the test. Later, we'll add more types here.